components/zenFunctions/init_component.sqf

```diff
@@ -44,4 +44,20 @@ if !(isNil 'zen_custom_modules_fnc_register') then
     ["[CA2] Respawners", "Respawn Random Player (OPFOR)", _spawnerEast] call zen_custom_modules_fnc_register;
     ["[CA2] Respawners", "Respawn Random Player (INDFOR)", _spawnerIndfor] call zen_custom_modules_fnc_register;
     ["[CA2] Respawners", "Respawn Random Player (Civilian)", _spawnerCiv] call zen_custom_modules_fnc_register;
+
+    _woundUnit = {_this#1 call f_fnc_zen_woundUnitRandomly};
```

Comment:
I'd note the first comment by Dedmen on this page:
https://community.bistudio.com/wiki/params

Doing it this way is fine because we know in this case that the type of `_this#1` is not an array, but in general you should put brackets around this so you don't have to do the "wait what type is this" arithmetic when debugging less straightforward code.

```suggestion
    _woundUnit = {[_this#1] call f_fnc_zen_woundUnitRandomly};
```